docs/generated/CLIOptions.rst

```diff
@@ -0,0 +1,168 @@
+--workers number		How many threads to use
+
+--dry-run		Skips real mutants execution. Disabled by default
```

Comment:
indentation?

Reply:
though you probably don't have control over this

Reply:
It's automatically generated. We shouldn't care about the prettiness of the contents as long as sphinx can handle it correctly.